.travis.yml

```diff
@@ -27,6 +27,10 @@ matrix:
       python: 3.7
     - env: TOXENV=py37-extra-deps
       python: 3.7
+    - env: TOXENV=py38
+      python: 3.8
+    - env: TOXENV=py37-extra-deps
+      python: 3.8
```

Comment:
I think we should keep `extra-deps` in the latest supported Python version, just as `pinned` is in the oldest. So I think this requires the following changes:
- Removing lines 28-29 (`py37-extra-deps` entry)
- Changing `py37-extra-deps` back to to `py38-extra-deps` in new line 32.

Reply:
My understanding was that the leveldb issue means that 3.8 is not officially supported yet, so shouldn't we leave it that extra dep check in 3.7 for now?